testing/trino-faulttolerant-tests/src/test/java/io/trino/faulttolerant/iceberg/TestIcebergParquetFaultTolerantExecutionConnectorTest.java

```diff
@@ -68,6 +68,13 @@ public void testStatsBasedRepartitionDataOnInsert()
         throw new SkipException("We always get 3 partitions with FTE");
     }
 
+    @Override
+    protected boolean isFileSorted(String path, String sortColumnName)
+    {
+        // This test produces many small files compared to the superclass. These files do not have enough data to test ordering using row group statistics.
+        throw new SkipException("Unimplemented");
```

Comment:
That's too late to skip the test: we already spent time writing data. 
(I'd rather throw Unsupported here, and override each of the tests to disable it -- if disabling is what i want)

Also, small file still can be sorted or not, so writing small files should prevent us from verifying sortedness.

BTW do you know why this class writes many small files? in its setup i don't see anything about it (did i miss?), so it feels like FTE implicitly results in many small files, implying FTE is not ready for a show yet (many small files is a big problem!)
cc @losipiuk @arhimondr

Reply:
> Also, small file still can be sorted or not, so writing small files should prevent us from verifying sortedness.

We could. In the other classes I'm just checking row group min/max values but we could read each row

Reply:
> BTW do you know why this class writes many small files?

I do not